endpoints.json

```diff
@@ -23,10 +23,25 @@
           "created_at": "2018-05-30T15:59:13.341Z",
           "votes": 0,
           "article_id": 1,
-          "article_img_url": "http://google.com"
+          "article_img_url": "http://google.com",
           "comment_count": 6
         }
       ]
     }
+  },
+
+  "POST /api/articles/:article_id/comments" : {
+    "description": "creates a comment for an article and returns the comment as a response",
+    "queries": [],
+    "exampleResponse": {
+      "comment": {
+        "comment_id": 304,
+        "body": "great cooking content",
+        "article_id": 1,
+        "author": "tickle122",
+        "votes": 0,
+        "created_at": "2023-11-20T17:01:34.992Z"
+      }
+    }

```

Comment:
Great to see the endpoints.json file updated with this new endpoint as well, but this is lacking some important info. This is a post request but it does not include what we expect the user to send as the new comment to actually post.